### ISSUE TITLE
Natural Earth appears to require JRE 1.8

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#1077]
I delivered a test version with NE today.  I also delivered a version last week.  Last week's version wouldn't work:  Debrief opened, but there was no Debrief content in there:
* no Tactical Analysis perspective
* DPF files weren't recognised
* DPF files couldn't open

The error log just contained a message saying: "No editor descriptor for id 'org.mwc.debrief.PlotEditor"  (or something like that).

So, for today I did a fresh build, and triple-checked it on my own machine.  Today's build wouldn't work on the client machines.

After a couple of hours of diagnosis it now appears that the NE builds require JRE 1.8.  On a JRE 1.7 machine I get the exact same errors.  When I upgrade the machine to 1.8, Debrief works perfectly.

I suspect this is down to the 1.8 requirement for GeoTools.  I've asked the clients to upgrade their JRE.

But would you trying to independently trying to verify this on one of your machines?  I'm keen to test my understanding.

**Comment by IanMayo** [comment]
Hmm, testing another VM bought up another solution.

The 64bit version seems to run on a 1.7 64bit  VM. That's confusing.

**Comment by pecko** [comment]
No one plugin requires Java 1.8. It should be run on JDK 1.7. 
The geotools plugin requires Java 1.6.
You have to use 64-bit JDK for 64-bit distribution and 32-bit JDK for 32-bit distribution.

@IanMayo What JDK you are using when building?

> "No editor descriptor for id 'org.mwc.debrief.PlotEditor"

I have faced this issue when geotools plugin isn't exported its packages (fixed with https://github.com/pecko/debrief/commit/06943e2c9493f3d942180246688a0a75cc5d9f1e)

Checking locally ...

**Comment by IanMayo** [comment]
@pecko  - I gave them at 32 bit distribution, 32 bit Java is their current *baseline*

Here's what my user reported
> I was on Java 6, Kev installed Java 8 32 bit, the Natural Earth version of Debrief opened and I had my plot, but it didn’t look the same it was squashed together north /south and elongated east/ west. 
 
@pecko  - Did I mention the fact that they wouldn't like the new projection?   :laughing:

**Comment by pecko** [comment]
We have changed base projection, so it is possible, but it isn't related to any JDK.

**Comment by IanMayo** [comment]
Sure.  Debrief w/ NE failed to start under JRE 6.  I started under JRE 8.  Maybe my client has just established that it has a requirement for JRE7.

I know the squashed data isn't related to the JRE - it's our new projection, as I warned you :laughing:

**Comment by pecko** [comment]
>  it's our new projection, as I warned you

I think that old projection (EPSG:3395) solves the case when there is TIFF image in this projection and there isn't *prj file. However, it isn't correct to use it for all layers. 
Maybe I wrong, but I think that we have correct projection now. 

RE JDK will test. Maybe some plugin requires JDK 1.7 ...

**Comment by pecko** [comment]
NE distribution doesn't work with JDK 1.6 because geotools libraries are compiled with Java 1.7.
We need minimal JDK 1.7 currently.
It should be work with JDK 1.8, but it isn't required

@IanMayo if you mean that we would support JDK 1.6 we can try to recompile geotools with JDK 1.6

**Comment by IanMayo** [comment]
No, we don't need 1.6 compatibility. We now understand the requirements. Is it possible for Debrief to " fail gracefully" if Debrief is run under an incompatible JRE?

A suitable error message would be better than starting but being inoperable.

**Comment by pecko** [comment]
I think, it is possible. Checking ...

**Comment by pecko** [comment]
I have added the commit: https://github.com/pecko/debrief/commit/65d1fae7c2a97c7e1b5bda37047eca5c4310ff75

Now, the geotools and o.m.d.core plugins require Java 1.7
I have reverted geotools Java reuirement to 1.7 because this plugin really requires 1.7.

Also, tycho compiles the plugins using source/target=1.7

The Debrief will not be able to start with Java 1.6. However, instead of some error or exception, it just freeze. Have to check more ...

The Debrief works fine with Java 1.7/1.8.
